web/templates/typeahead_list_item.hbs

```diff
@@ -35,9 +35,7 @@
     {{> status_emoji status_emoji_info}}
     {{~/if}}
     {{~#if has_secondary}}
-    <small class="autocomplete_secondary">
-        {{~ secondary ~}}
-    </small>
+    <small class="autocomplete_secondary rendered_markdown">{{{secondary}}}</small>
```

Comment:
We should introduce a new variable here `has_rendered_markdown_secondary` which will toggle between new and the old version of it is rendered.

Reply:
Can you call that field `secondary_html`? To prevent XSS vulnerabilities, we want to make it really visually obvious what values will be inserted into the DOM as HTML, and that naming convention really helps with this.

We might want to do a follow-up audit of `git grep }}} web/templates/` to rename every other field that's being passed without this convention; it looks like there's a handful.

This also suggests a cleaner way to name `has_rendered_markdown_secondary`; it can just be `has_secondary_html`.